### PR TITLE
[Hotfix] Sort available plans

### PR DESCRIPTION
**Pull request by murtaza-swati**
[Hotfix] Sort available plans

### CODE REVIEW

**Review by stanjaromin — APPROVED** [review]
LGTM

File: app/components/billing/select-plan.js [hunk 1]
```diff
@@ -133,6 +133,14 @@ export default Component.extend({
     );
   },
 
+  customSort(plans) {
+    const planIds = ['starter_plan', 'monthly_usage_plan_10k_credits', 'annual_usage_plan_130k_credits'];
+    const specificPlans = plans.filter(plan => planIds.includes(plan.id));
+    const otherPlans = plans.filter(plan => !planIds.includes(plan.id));
+
+    return [...specificPlans, ...otherPlans];
+  },
```
Comment (DominikAlberski) [inline app/components/billing/select-plan.js:142]: I would sort by price if it is available

**Review by DominikAlberski — APPROVED** [review]
LGTM

File: app/components/billing/select-plan.js (hunk 1 above)
Comment (murtaza-swati) [reply app/components/billing/select-plan.js]: Thanks - yes - startingPrice was available.
changed the logic

**Review by DominikAlberski — APPROVED** [review]

**Pull request merged**